recipes/rechunker/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "rechunker" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 56377a380acad90a87d3482dd6b7851e557bd0d86bfbcd5de874205b8e213934
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6.0
```

Comment:
3.6 is good enough, no need to specify the micro version.

Reply:
Thanks, good to know for next time!